src/thinblock.cpp

```diff
@@ -1817,7 +1818,8 @@ void BuildSeededBloomFilter(CBloomFilter &filterMemPool,
     if (nTimePassed > nHoursToGrow * 3600)
         nFPRate = nMaxFalsePositive;
 
-    filterMemPool = CBloomFilter(nElements, nFPRate, insecure_rand(), BLOOM_UPDATE_ALL);
+    uint32_t nMaxFilterSize = std::max(MAX_BLOOM_FILTER_SIZE, pfrom->nXthinBloomfilterSize);
```

Comment:
this means that it's not possible to set a value lower than `MAX_BLOOM_FILTER_SIZE`, while I think this is right I guess if we shouldn't print a warning in the log or in the console in case that some try to set a value lower than this.

Reply:
You bring up a good point, although I don't think it makes sense right at this point to print out a log message because here we are building the get_xthin message.  We don't have any control over the pfrom->nXthinBloomFilterSize so printing out a warning or message will potentially just fill up the log file with messages every time a block is found and for which we can do nothing about.  However, we could print out a warning, if for instance, someone mis-configures bitcoin.conf with a very small bloom filter size...so on startup we could print that out once in the log..

Reply:
agreed